doc/howto/images_copy.md

```diff
@@ -6,3 +10,91 @@
 
   - local file
   - file on web server
+
+
+### Direct pushing of the image files
+
+This is mostly useful for air-gapped environments where images cannot be
+directly retrieved from an external server.
+
+In such a scenario, image files can be downloaded on another system using:
+
+    lxc image export ubuntu:22.04
+
+Then transferred to the target system and manually imported into the
+local image store with:
+
+    lxc image import META ROOTFS --alias ubuntu-22.04
+
+`lxc image import` supports both unified images (single file) and split
+images (two files) with the example above using the latter.
+
+### File on a remote web server
+
+As an alternative to running a full image server only to distribute a
```

Comment:
In this context what is a "full image server"? Is this a LXD server hosting just images?

If so, we could say something like "a LXD server only to distribute images to users"?